server/helper/mailer.js

```diff
@@ -6,7 +6,7 @@ const handlebars = require('handlebars');
 const EmailTemplate = require('email-templates').EmailTemplate;
 const path = require('path');
 
-const secret = require(path.resolve('./config', 'mailerConfig.json'));
+const secret = require('./../../config/mailerConfig.json');
```

Comment:
if i stay this in previously stated it will generate a warn when our application will be boot..i can change this if it is not so big deal, does it?